### DISCUSSION TITLE
Document null value without quotations in OpenApi request body.

### DISCUSSION THREAD

**Discussion by PabloRuizCuevas** [fastapi/fastapi#8490] [Questions]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from fastapi import FastAPI
from pydantic import BaseModel
from typing import Optional
from uuid import UUID
import uvicorn


class Table(BaseModel):
    # ID: Optional[UUID]      # the docs show a example UUID, ok
    ID: Optional[UUID] = None # the docs still shows a uuid, when it should show a null or valid None value.

app = FastAPI()  
    
@app.post("/table/", response_model=Table)
def create_table(table: Table):
    # here we call to sqlalchey orm etc.
    return 'nothing important, the important thing is in the docs'
    
if __name__ == "__main__":
    uvicorn.run(app, host="0.0.0.0", port=8000)
```

### Description

Using a ORM, I want to do a POST request letting some fields with a null value, which will be translated in the database for the default value specified there.

The problem is that OpenApi (Swagger) docs, ignores the default None in the schema example (request body)  and still prompts a UUID (or other type) by default.

So we get  in the example:

```
{
 "ID": "3fa85f64-5717-4562-b3fc-2c963f66afa6"
}
```

Instead of this:

```
{
 "ID": null, # null is the equivalent of None here, notice it doesn't have quotation marks
}
```
Regardless if the table says `ID: Optional[UUID]` or `ID: Optional[UUID] = None`, this behavior is not the same when you have a string for example, where the one documented will be the one you put as the default value.

The null without quotation marks is important, a string "null" won't be valid for this case.

### Operating System

Windows

### FastAPI Version

0.77.1

### Python Version

3.9

### Additional Context

I already made a question in SO [here](https://stackoverflow.com/questions/72214347/how-to-handle-default-none-null-in-fastapi-openapi-swagger?noredirect=1#comment127597828_72214347)

@JarroVGIT

**Comment by Cajuteq** [comment]
Hi, 

first it seems the null values were ignored in swagger until recently [swagger issue](https://github.com/swagger-api/swagger-inflector/issues/371) 

So I think one could try to pass arguments using fastapi [swagger_ui_parameters](https://fastapi.tiangolo.com/advanced/extending-openapi/#configuring-swagger-ui) but I didn't found swagger args related to the null examples handling so maybe one of us will need to add the parameter in swagger ui.

second there seems to be a nullable ( x-nullable) field in OpenAPI V3 (V2) so we might be able to leverage this by modifying the openapi schema directly in fastapi by adding nullable at the right place following this [example](https://fastapi.tiangolo.com/advanced/extending-openapi/#modify-the-openapi-schema)

I don't think I will take the time to test either anytime soon, but those seems to be strong starting points imho
